### DISCUSSION TITLE
Youtbe videos in smplayer start and stop

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#258] [Q&A]
I'm using version 2180 rev 9887 64bit. Youtube videos entered via "open", "URL" start and stop and mostly just stop. It goes for 1 sec then stops for a few then repeats. If I try to move forward or back it really stops for some time. I'm using win10. It used to play these quite well but no longer. Any suggestions? Thanks.

**Comment by smplayer-dev** [comment]
The solution is here: https://github.com/smplayer-dev/smplayer/discussions/247

**Reply by tack32** [reply]
So far that hasn't worked. Before reading this I tried the latest update but still developmental. That seems to have made it worse. Now it simply says something went wrong and doesn't play youtube videos at all.?
![youtube settings and version](https://user-images.githubusercontent.com/93066797/138937511-f4bba55e-8f83-4105-af29-f0ccd4fac75f.jpg)

**Reply by tack32** [reply]
btw this is is the mpv log from the last time I tried a Youtube video. Thanks.
[mplayer log youtube crash or buffer.txt](https://github.com/smplayer-dev/smplayer/files/7420643/mplayer.log.youtube.crash.or.buffer.txt)
o.

**Reply by smplayer-dev** [reply]
I can't reproduce this problem with the [latest version](https://github.com/smplayer-dev/smplayer/releases/tag/v21.8.0.9988).

**Reply by tack32** [reply]
I installed https://github.com/smplayer-dev/smplayer/releases/tag/v21.8.0.9988 
and also https://sourceforge.net/projects/smtube/files/SMTube/21.7.0/smtube-21.7.0-x64.exe/download
but still get "MPlayer/mpv has finished unexpectedly. More info in the log" This is the log if it offers any clues. Thanks
[log smplayer youtube crash.txt](https://github.com/smplayer-dev/smplayer/files/7428237/log.smplayer.youtube.crash.txt)

**Reply by smplayer-dev** [reply]
Did you download yt-dlp with the option Help-> Install / Update youtube code?

**Reply by tack32** [reply]
Sorry, could you say what that link might be (win10 64 bit)? Thanks

**Reply by smplayer-dev** [reply]
In smplayer, open the Help menu, there's an option there to install the "youtube code" . That will download and install yt-dlp.

**Reply by tack32** [reply]
Yes, it's working now! Thanks!
